src/node/cli.ts

```diff
@@ -19,10 +21,11 @@ export enum LogLevel {
 export class OptionalString extends Optional<string> {}
 
 export interface Args extends VsArgs {
+  readonly config?: string
   readonly auth?: AuthType
+  readonly password?: string
```

Comment:
So I only really added this here to avoid having different parameters in flags vs the config file. We can just error out if we see `--password` in the cli flags.

Reply:
Means more docs as well.

Reply:
Oooh yeah I see what you mean. That's a tough one. Even if we error it would show in `--help` but then you wouldn't be able to specify it. But I suppose if we're really clear about it in the error then it won't be a problem.

Reply:
Yeah it's definitely nice to say "just look at --help" for what you can put in the config instead of having a separate doc for it.

Reply:
Yea just for now until we use a CLI library or do the Go thing.